crossplane-infra-access/README.md

```diff
@@ -0,0 +1,50 @@
+# Crossplane infrastructure access
+
+## Intro
+
+Team Honeybadger is deploying Crossplane to MCs. Crossplane can be used to manage infrastructure on the cloud provider side by both us and out customers.
+
+In order to do that, Crossplane has to be able to authenticate and authorize against the infrastructure provider. The access model that Crossplane uses depends on the user. By default, Crossplane doesn't require nor provide any access info. The administrator of Crossplane can create as many access secrets as he/she wants and can grant access to them to other Kubernetes users using the standard RBAC configuration.
+
+Example:
+1. Crossplane admin logins to an AWS account. In IAM, it creates two roles:
```

Comment:
Who is Crossplane admin? a person or a tool/operator?

Reply:
I think a person is meant here. Could be GS staff or customer admins too. We need to manage the scope of the access keys / tokens to avoid exposing full access in case of an issue with the secrets.

Reply:
Yes, that's a person that is responsible for configuring crossplane for users, either internal or external